doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,241 @@
+| Location     | ``plutus-apps/plutus-ledger-constraints``                     |
++--------------+---------------------------------------------------------------+
+
+`Plutus ledger constraints <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-ledger-constraints>`_ contains a constraints-based API that can be used to generate on-chain validation functions and to build ransactions by providing a list of constraints. 
+The main design is to be able to use the same constraints on-chain and off-chain in a plutus application. 
+The off-chain part generates transactions based on types in `Plutus ledger <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-ledger>`_. 
+
+For example:
+
+   * ``checkScriptContext (MustSpendAtLeast 10Ada, MustProduceOutput myOutput, …)``
+   * ``mkTx (MustSpendAtLeast 10Ada, MustProduceOutput myOutput, …)``
+
+12. Plutus contract state machine
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+---------------------------------------------------------------+
+| Plutus contract state machine                                                |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-contract``                               |
++--------------+---------------------------------------------------------------+
+
+`Plutus contract state machine <https://github.com/input-output-hk/plutus-apps/blob/main/plutus-contract/src/Plutus/Contract/StateMachine.hs>`_ is a library that is a useful high-level tool for defining and modeling a Plutus application (smart contract) based on the State Machine formalism. 
+It is helpful for writing a reference implementation for testing before creating the production version. 
+However, we do not recommend using it in production as the scripts are too big to run on-chain. 
+
+13. Plutus use case examples
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+---------------------------------------------------------------+
+| Plutus use case examples                                                     |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-use-cases``                              |
++--------------+---------------------------------------------------------------+
+
+`Plutus use case examples <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-use-cases>`_ contains hand-written examples for the use cases we currently have. 
+The primary examples are: 
+
+   * Auction, 
+   * Crowdfunding, 
+   * Game, 
+   * GameStateMachine, and 
+   * Vesting. 
+
+For each plutus application use case, we provide test scenarios (or test cases) with and without the :hsmod:`Plutus.Contract.Test.ContractModel`.
+
+The examples are for testing and educational purposes. 
+While they work in the `plutus-contract emulator`, they are not guaranteed to work on the actual Cardano network. 
```

Comment:
Add the following: [...] they are not guaranteed to work on the actual Cardano network. The main reason is that the size of the produced Plutus scripts are too big to fit in a transaction given current protocol parameters.